hometask2/controllers/userController.js

```diff
@@ -0,0 +1,102 @@
+import uuid from 'uuid-random';
+
+const usersMap = new Map();
+
+//read
+exports.read = (req, res) => {
```

Comment:
Consider the following:
1. `exports.smth` is just a syntactic sugar for `module.exports = { smth ....}` which is part of CommonJS Module System - use ES6 Imports/Exports Module System instead, either `export default smth;` or `export { smth ...};`
2. Keep all the imports at the top of the file and all the exports at the bottom. So in the case of the current line of code, you would have `const read = (...) => {...};` and at the bottom of the file `export { ..., read, ....};`